iphone/CoreApi/CoreApi/Traffic/MWMMapOverlayManager.mm

```diff
@@ -158,4 +181,29 @@ + (void)setIsoLinesEnabled:(BOOL)enable {
   f.SaveIsolinesEnabled(enable);
 }
 
++ (void)setOutdoorEnabled:(BOOL)enable {
+  if (enable) {
+    [self setTransitEnabled:false];
+    [self setTrafficEnabled:false];
+  }
+
+  auto &f = GetFramework();
+  switch (f.GetMapStyle()) {
+    case MapStyleClear:
+    case MapStyleVehicleClear:
+    case MapStyleOutdoorsClear:
+      f.SetMapStyle(enable ? MapStyleOutdoorsClear : MapStyleClear);
+      break;
+    case MapStyleDark:
+    case MapStyleVehicleDark:
+    case MapStyleOutdoorsDark:
+      f.SetMapStyle(enable ? MapStyleOutdoorsDark : MapStyleDark);
+      break;
+    default:
+      break;
+  }
+
+  [NSNotificationCenter.defaultCenter postNotificationName:didChangeOutdoorMapStyle object:nil];
```

Comment:
If it's a temporary solution, a TODO: with how it should be rewritten would be helpful.